website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -36,6 +36,8 @@ Make sure to define `IMG` when you call `make`:
   make docker-build docker-push IMG=$OPERATOR_IMG
   ```
 
+**Note**: You need [kube-builder][kube-builder] installed as the `docker-build` target uses it for the controller test suite.
```

Comment:
You definitely don't need kubebuilder installed, but you do need envtest binaries installed, which should be done by `make test` for you. Which operator-sdk version are you using?

Reply:
Built it from master before the holidays:

```
$ operator-sdk version
operator-sdk version: "v1.2.0-31-ge4efaa6b", commit: "e4efaa6bea37e99dcd3e8aa8a27f5fcacb30d5be", kubernetes version: "v1.19.4", go version: "go1.15.6", GOOS: "linux", GOARCH: "amd64"
```

I didn't see any mention of running `make test` in https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/ or is it called implicitly? I did however get it working by installing `kubebuilder` so that is why I put it down.

Reply:
Hi @hickeyma, 

Also, note that this is the **Ansible** quick start. You cannot use a controller test suite for ansible at all. The makefile target will only use the **docker** instruction to build the image and this error would not show up. See: https://github.com/operator-framework/operator-sdk/blob/master/testdata/ansible/memcached-operator/Makefile#L41-L42

Reply:
ok, so can remove it from here. I am using the Go tutorial and thought it might be needed in the Ansible tutorial as well.

Reply:
@camilamacedo86 - I'm trying to get started with an Ansible operator following it's tutorial and getting:

```
$ IMG=ttl.sh/example-memcached:1h make docker-build docker-push
env: Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/jgeerling/.rvm/bin:/Users/jgeerling/Downloads/memcached-operator/bin: No such file or directory
env: Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/jgeerling/.rvm/bin:/Users/jgeerling/Downloads/memcached-operator/bin: No such file or directory
docker build . -t ttl.sh/example-memcached:1h
env: Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/jgeerling/.rvm/bin:/Users/jgeerling/Downloads/memcached-operator/bin: No such file or directory
make: *** [docker-build] Error 127
```

Not sure where that's coming from though :/

Reply:
Removed.